base/allocator/partition_allocator/partition_alloc_base/threading/platform_thread.h

```diff
@@ -18,7 +18,9 @@
 #include "base/allocator/partition_allocator/partition_alloc_base/time/time.h"
 #include "build/build_config.h"
 
-#if BUILDFLAG(IS_WIN)
+#if defined(STARBOARD)
```

Comment:
This is still getting included even though we're not building with partition_alloc?